README.md

```diff
@@ -115,6 +96,6 @@ others in the community.
 If you want to contribute, you're most welcome!  There are *lots* of
 ways to contribute, regardless of your Haskell background.  For
 example, even someone with no Haskell experience whatsoever could
-still help with *e.g.* game design, playtesting, and level design for
-challenge mode.  Check out the [CONTRIBUTING](CONTRIBUTING.md) file
-for more specific information about how to contribute.
+still help with *e.g.* game design, playtesting, and creating
+challenge scenarios.  Check out the [CONTRIBUTING](CONTRIBUTING.md)
```

Comment:
```suggestion
challenges and scenarios.  Check out the [CONTRIBUTING](CONTRIBUTING.md)
```

Reply:
I mean, challenges are a type of scenario... but people reading this document don't know that.  So I guess this is better. :smile: